### DISCUSSION TITLE
MD: How to keep a methanol solute centered in a 40-water cluster?

### DISCUSSION THREAD

**Discussion by johnwkeller54** [grimme-lab/xtb#1036] [General]
I used --md --alpb water, plus $md and  $wall sections.  Setting beta=3,6,or 12 seems to have very little effect. The waters migrate apart and the solute ends up on the periphery. 
A typical input file is attached.
John Keller
[meoh-40water.txt](https://github.com/grimme-lab/xtb/files/15299894/meoh-40water.txt)

**Comment by awvwgk** [comment]
You can try two walls, one only for the solute to keep it centered and one for the whole cluster to avoid that the solvent moves away around the fixed solute.

**Comment by johnwkeller54** [comment]
HI Sebastian- Thanks for the tip!
John Keller

**Comment by johnwkeller54** [comment]
OK - does the term following the comma in the "sphere" line refer to which species is limited by the wall? 
Is this an atom list, or the frament number as defined in the "molecular fragmentation" section under Calculation Setup?

```
-----------------------
molecular fragmentation (1/2 indicates fragments):
111111222222222222222222222222222222222222222222222222222222222222222222
222222222222222222222222222222222222222222222222222222222222222222222222
222222222222222222222222222222222222222
# atoms in fragment 1/2:     6   177
 fragment masses (1/2) :       32.04     1062.90
CMA distance (Bohr)    :   0.928
constraining FC (au)   :  1.0000
```
-----------------------

```
$wall
   potential=logfermi
   sphere: 6.0, 1
   sphere: 30.0, 2
$end
```

John

**Reply by awvwgk** [reply]
The argument for this is a list of atom indices and unrelated to the fragment printout.

**Comment by johnwkeller54** [comment]
$CMA  - Would this command be useful in keeping molecules centered?
 Is this entered as a stand-alone statement, or does it require $end ?
 Does it take any parameters or arguments?

**Reply by awvwgk** [reply]
This only modifies the input at the start of xtb, it has no effect during the MD.
